etc/telegraf/telegraf.toml.j2

```diff
@@ -23,6 +23,26 @@
     hostname = "{{ hostname }}"
     omit_hostname = false
 
+# Integration usage monitoring
+# For each type of integration shell scripts should return 1 if it's enabled, 0 otherwise.
+{% for integration in integrations %}
```

Comment:
From the beginning, I thought that we can have separate processes for each metric that we wanna push so I implemented this way. For now, it's named as integrations since that's the only use case for now but in the future, it can be anything actually. To be able to push any metric, we need a path to the executable and the name of the metric (which I kept the same as the executable file name).

For integrations, another approach can be like the following. We can check is_enabled for each integration in telegraf.py and pass a dictionary to telegraf where the name of the integrations are keys and 0 or 1 are the values. 
But yeah, I just thought separate executables can be more extendable in the future. And also, we can change to pyhton as well, let's decide if we'll have executables or not first :)  

Also, I find it valuable to push 0 if integrations are disabled so that we can see that we are checking but they are disabled. So, I'm not in favor of only pushing when it's enabled kind of approach. Not sure you mean that but just wanted to clarify my approach to this :)

Reply:
Hey @pappachino, refactored code to use the existing methods